AWSCognitoIdentityProvider/AWSCognitoIdentityUserPool.h

```diff
@@ -343,6 +344,11 @@ typedef NS_ENUM(NSInteger, AWSCognitoIdentityClientErrorType) {
  */
 -(id<AWSCognitoIdentityCustomAuthentication>) startCustomAuthentication;
 
+/**
+ Initialize ui to prompt end user for custom authentication flow. This is added to support AWSMobileClient.
+ */
+-(id<AWSCognitoIdentityCustomAuthentication>) startCustomAuthentication_v2;
```

Comment:
Move to internal

Reply:
Not sure how to achieve this. Should I create a new protocol that extend on this?

Reply:
Added an internal protocol.